frontend/configs/dev/appConfigMap.js

```diff
@@ -534,7 +543,7 @@ export const configAppMap = {
           layout: {
             // visibility: 'none' ,
           },
-          paint: fillPaint,
+          paint: fillPaintBis,
```

Comment:
maybe rename to `fillPaint`

Reply:
bein en fait j'utilise les 2 : `fillPaint` pour les régions et `fillPaintBis` pour les départements ... c'est pour le moment juste une opacité différente entre les 2 layers

Reply:
Ok, trouver un meilleur non alors ;-)